test/JDBC/sql_expected/BABEL-3512.out

```diff
@@ -0,0 +1,763 @@
+
```

Comment:
Do we need two output files?

Reply:
no only one, will remove the redundant file

Reply:
Please remove this file